xz/tests/test.bats

```diff
@@ -0,0 +1,5 @@
+@test "Version matches plan" {
+  TEST_PKG_VERSION="$(echo "${TEST_PKG_IDENT}" | cut -d/ -f3)"
+  actual_version="$(hab pkg exec "${TEST_PKG_IDENT}" xz --version | awk '/XZ Utils/{print $0}')"
```

Comment:
It might be a bit more straight forward to `print $4` which will be the version number. Then you could do a simple comparison on the next line.

Reply:
Amended